### PR TITLE
Make improvements to MARC import

### LINKED ISSUE #1426: Feedback Beta Version: Import mrk records
1. New field that new imported records will be tagged with on the import action, so as to monitor and be easily identifiable when searching. It can be a field assigned on the backend on import or a specific metadata field with a value.

2. If no record is selected and then click "Submit Selected Records" the Message that should be shown is: No record was selected and imported!

3. Not clear heading **Unmatched heading** -> use Heading doesn’t exist

4. I receive Fatal: 191$b Ambiguous heading: E/ESCWA/
. Additional specification should probably be set.

Authority 191 $bE/ESCWA/
exists, so I shouldn't have received the "Ambiguous heading". This is a valid value just with the b!
This occurs because we have additional c value like for example in the following:
E/ESCWA/
E/ESCWA/; 26
E/ESCWA/; 25
->No message "Ambiguous heading"/ Please adjust code to select the valid value (one without c). If it has a c field then the one with b and c.

5.New auths can be imported - Expected behavior and user stories need to be identified and discussed - no specs.
I imported a duplicate one -no message received, created a bib with the link to its duplicate
1373594
887673

### PR DESCRIPTION
**Pull request by aaronhelton**
Closes #1426

### CODE REVIEW

**Comment by aaronhelton** [comment]
@jbukhari the main improvements left here are from items 4 and 5. We need to make sure that messaging about ambiguous headings is accurate, and we need to make sure that duplicate auth records aren't created.

**Review by jbukhari — COMMENTED** [review]
This pending dlx functionality will resolve the authorities validations for data that has the auth xref in subfield $0 https://github.com/dag-hammarskjold-library/dlx/issues/366

**Comment by jbukhari** [comment]
> This pending dlx functionality will resolve the authorities validations for data that has the auth xref in subfield $0 [dag-hammarskjold-library/dlx#366](https://github.com/dag-hammarskjold-library/dlx/issues/366)

Implemented as of dlx v1.4.17 https://github.com/dag-hammarskjold-library/dlx/releases/tag/v1.4.17

**Comment by aaronhelton** [comment]
Presently, we can export and import mrk records. We can export CSV and XML.

File: dlx_rest/static/js/import.js [hunk 1]
```diff
@@ -205,23 +234,30 @@ export let importcomponent = {
...
                                     if (subfield.xref instanceof Error) {
                                         // unresolved xrefs are set to an Error object
-                                        fatalErrors.push({"message": `Fatal: ${field.tag}$${subfield.code} ${subfield.xref.message}: ${subfield.value}`})
+                                        fatalErrors.push({ "message": `Fatal: ${field.tag}$${subfield.code} ${subfield.xref.message}: ${subfield.value}` })
                                     }
                                 }
                             }
 
-                            this.records.push({"jmarc": jmarc, "mrk": mrk, "validationErrors": validationErrors, "fatalErrors": fatalErrors, "checked": false})
+                            // Set a field indicating the record was imported
```
Comment (jbukhari) [inline dlx_rest/static/js/import.js:253]: Are we sure we want to set this as a value inside the record data?

File: dlx_rest/static/js/jmarc.mjs
Comment (jbukhari) [inline dlx_rest/static/js/jmarc.mjs:1]: Do we really need to make all these changes to the spacing at this time? Is there a way to see only the changes related to parsing the MRK?

Comment (aaronhelton) [reply dlx_rest/static/js/jmarc.mjs:1]: Apologies. I have automatic formatting on save so it does this for any js files I'm working on.

Comment (aaronhelton) [reply dlx_rest/static/js/jmarc.mjs:1]: Mainly here: https://github.com/dag-hammarskjold-library/dlx-rest/pull/1515/files#diff-de7e9e3a2adcaf0e749f93600300cdfeffb2ea72de7222d592a62074b7e81d0dR765

File: dlx_rest/static/js/import.js (hunk 1 above)
Comment (aaronhelton) [reply dlx_rest/static/js/import.js:253]: The request was to use a field to indicate the record was imported. We can present it and see if that suffices.

**Review by jbukhari — APPROVED** [review]

**Pull request merged**